azure-toolkit-libs/azure-toolkit-auth-lib/src/main/java/com/microsoft/azure/toolkit/lib/auth/AzureTokenCredentialsAdapter.java

```diff
@@ -10,25 +10,24 @@
 import com.azure.core.credential.TokenRequestContext;
 import com.microsoft.azure.AzureEnvironment;
 import com.microsoft.azure.credentials.AzureTokenCredentials;
-
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 
 /**
  * Convert token credential in azure-identity to legacy AzureTokenCredentials
  * Refers https://github.com/jongio/azidext/blob/master/java/src/main/java/com/azure/identity/extensions/AzureIdentityCredentialAdapter.java
  */
-public class AzureIdentityCredentialTokenCredentials extends AzureTokenCredentials {
+public class AzureTokenCredentialsAdapter extends AzureTokenCredentials {
```

Comment:
add a static `fromTokenCredential` method in this class and then `AzureTokenCredentialsConverter` can be removed